listenbrainz/webserver/static/js/jsx/lastFmImporterModal.jsx

```diff
@@ -0,0 +1,120 @@
+'use strict'
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Importer from './importer'
+import { faTimes } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+class Modal extends React.Component {
```

Comment:
This Modal should live in a different jsx file and should be called `LastFmImporterModal`